build-system/tasks/performance/compile-scripts.js

```diff
@@ -28,6 +29,7 @@ async function compileScripts(urls) {
   if (!argv.nobuild) {
```

Comment:
--nobuild is only used when you have already built the binary and don't want to build it again. 
There's no need to await for `vendorConfigs`.